.rubocop.yml

```diff
@@ -82,6 +82,9 @@ RSpec/MultipleExpectations:
 RSpec/NestedGroups:
   Max: 3
 
+RSpec/NamedSubject:
```

Comment:
It's not a terrible cop this one. We enforce it in some projects. Maybe we should here too.